spec/services/cleaners/advocate_claim_cleaner_spec.rb

```diff
@@ -12,102 +48,72 @@
   describe '#call' do
     subject(:call_cleaner) { cleaner.call }
 
-    context 'with an "Appeal against sentence" case type including a fixed fee' do
-      let(:claim) do
-        create(:draft_claim, case_type: create(:case_type, :appeal_against_sentence), fixed_fees: fixed_fees)
-      end
-      let(:fixed_fees) { [build(:fixed_fee, :fxase_fee, :with_date_attended, rate: 9.99)] }
+    let(:claim) do
+      build(
+        :draft_claim,
```

Comment:
for clarity I would use
:advocate_final_claim, :draft

The older `<state>_claim` factories should be considered deprecated as they pre-date the existence of any  other claim type (i.e. when there was only one claim type) and are therefore not very descriptive